src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionJavaPatchFilter.java

```diff
@@ -241,7 +240,9 @@ private void loadPatchFile() throws CheckstyleException {
                 filters.add(element);
             }
         }
-        catch (IOException ex) {
+        // -@cs[IllegalCatch] There is no other way to deliver filename that was under
+        // processing when a jgit exception occurs.
+        catch (Exception ex) {
             throw new CheckstyleException("an error occurred when load patch file", ex);
```

Comment:
As we touch this, please make variable to be `exception `
And print path to variable

Reply:
Both done.